conans/test/build_helpers/autotools_configure_test.py

```diff
@@ -564,13 +564,119 @@ def cross_build_command_test(self):
         ab.configure(target="i686-apple-darwin")
         self.assertEquals(runner.command_called, "./configure  --target=i686-apple-darwin")
 
-    def test_make_targets(self):
+        conanfile = MockConanfile(MockSettings({"build_type": "Debug",
+                                                "arch": "x86",
+                                                "os": "Windows",
+                                                "compiler": "gcc",
+                                                "compiler.libcxx": "libstdc++"}),
+                                  None, runner)
+        ab = AutoToolsBuildEnvironment(conanfile)
+        ab.configure()
+        self.assertIn("i686-w64-mingw32", ab.host)
+        if platform.system() == "Windows":
+            self.assertIn("x86_64-w64-mingw32", ab.build)
+            self.assertIn("./configure  --build=x86_64-w64-mingw32 --host=i686-w64-mingw32",
+                          runner.command_called)
+        elif platform.system() == "Linux":
+            self.assertIn("x86_64-linux-gnu", ab.build)
+            self.assertIn("./configure  --build=x86_64-linux-gnu --host=i686-w64-mingw32",
+                          runner.command_called)
+        else:
+            self.assertIn("x86_64-apple-darwin", ab.build)
+            self.assertIn("./configure  --build=x86_64-apple-darwin --host=i686-w64-mingw32",
+                          runner.command_called)
+        ab.configure(build="fake_build_triplet", host="fake_host_triplet")
+        self.assertIn("./configure  --build=fake_build_triplet --host=fake_host_triplet",
+                      runner.command_called)
+
+    def test_make_targets_install(self):
```

Comment:
Missing a test assigning `self.build, self.host, self.target ` in the  `ab` instance.